miden-tx/src/host/mod.rs

```diff
@@ -73,6 +98,9 @@ impl<A: AdviceProvider> Host for TransactionHost<A> {
         match event {
             AddAssetToAccountVault => self.acct_vault_delta_tracker.add_asset(process),
             RemoveAssetFromAccountVault => self.acct_vault_delta_tracker.remove_asset(process),
-        }
+            PushAccountProcedureIndex => self.on_push_account_procedure_index(process),
+        }?;
+
+        Ok(HostResponse::None)
```

Comment:
Could be unrelated. Made me think of another instance where you swapped HostResponse::None to Ok(()).

 In the account_delta.rs file of the host filder in miden-tx. 

 You removed the HostResponse. In the add_asset() and remove_asset() functions. Why removed there and kept here?

Reply:
We can't remove here because `Host` trait signature requires `HostResponse` to be returned. Though, we can of course, modify `Host` trait itself.